ksql-common/src/main/java/io/confluent/ksql/properties/with/CommonCreateConfigs.java

```diff
@@ -117,7 +119,14 @@ static void addToConfigDef(
             ConfigDef.Type.STRING,
             null,
             Importance.LOW,
-            "The fully qualified name of the Avro schema to use");
+            "The fully qualified name of the Avro schema to use"
+        )
+        .define(
+            VALUE_DELIMITER_PROPERTY,
+            ConfigDef.Type.STRING,
+            null,
+            Importance.LOW,
+            "The delimiter to use when VALUE_FORMAT='DELIMITED'.");
```

Comment:
nit: look to have similar help here as the docs so that we can (one day) generate the docs from this class.  e.g. include info on the special strings.